README.md

```diff
@@ -60,6 +60,7 @@ sudo apt install clang cmake doxygen graphviz g++ libfreetype-dev libglfw3-dev \
     libspdlog-dev python3-glad
 ```
 Then, clone and build NovelRT:
+> **WARNING:** we currently do not support GCC as a compiler option. Expect errors/bugs if you try to use it - consider clang instead.
```

Comment:
maybe instead of "consider" we could say "supported compilers are MSVC and clang for their respective platforms."?